### PR TITLE
Test all portfolio configs.

### PR DESCRIPTION
**Pull request by jendrikseipp**

### CODE REVIEW

File: driver/tests.py [hunk 1]
```diff
@@ -59,6 +61,42 @@ def test_portfolios():
...
+        config[index] = part
+    return config
+
+
+def test_portfolio_configs():
+    all_configs = set()
+    for portfolio in PORTFOLIOS.values():
+        configs = _get_portfolio_configs(Path(portfolio))
+        all_configs |= set(tuple(_convert_to_standalone_config(config)) for config in configs)
+    for config in set(all_configs):
```
Comment (FlorianPommerening) [inline driver/tests.py:96]: Do you need `set(...)` here?

Comment (jendrikseipp) [reply driver/tests.py:96]: Apparently yes.

Comment (FlorianPommerening) [reply driver/tests.py:96]: Why? Isn't `all_configs` a set already? It is not modified in the loop, so I don't see why you'd need a copy.

Comment (jendrikseipp) [reply driver/tests.py:96]: Ah, I thought you mean a different line. Thanks, I removed it.

**Pull request merged**